Makefile

```diff
@@ -2346,6 +2346,10 @@ rocksdbjavastaticdockers390x:
 	mkdir -p java/target
 	docker run --rm --name rocksdb_linux_s390x-be --attach stdin --attach stdout --attach stderr --volume $(HOME)/.m2:/root/.m2:ro --volume `pwd`:/rocksdb-host:ro --volume /rocksdb-local-build --volume `pwd`/java/target:/rocksdb-java-target --env DEBUG_LEVEL=$(DEBUG_LEVEL) evolvedbinary/rocksjava:ubuntu18_s390x-be /rocksdb-host/java/crossbuild/docker-build-linux-centos.sh
 
+rocksdbjavastaticdockerriscv64:
+	mkdir -p java/target
+	docker run --rm --name rocksdb_linux_riscv64-be --attach stdin --attach stdout --attach stderr --volume $(HOME)/.m2:/root/.m2:ro --volume `pwd`:/rocksdb-host:ro --volume /rocksdb-local-build --volume `pwd`/java/target:/rocksdb-java-target --env DEBUG_LEVEL=$(DEBUG_LEVEL) evolvedbinary/rocksjava:ubuntu20_riscv64-be /rocksdb-host/java/crossbuild/docker-build-linux-centos.sh
+
```

Comment:
I still need to create and publish a Docker image for musl riscv64, we can then add that in here... I will try and do this later today...

Reply:
@luhenry So I have found two possible Alpine Linux for riscv64 Docker Images in Docker Hub:
1. riscv64/alpine:20210804
2. riscv64/alpine:20230901
but neither of them have an OpenJDK package available. Any thoughts on an easy way to solve that?

Reply:
There is no support for musl for the OpenJDK on RISC-V AFAIK. And that would not be a trivial amount of work, especially since I don't know how much of Alpine is supported on RISC-V at the moment. RISC-V is very much a glibc environment at the moment (for better or worst)

Reply:
It seems that Alpine (at least C++/musl compiler wise) is working fairly well on RISC-V. With this Dockerfile https://github.com/evolvedbinary/docker-rocksjava/pull/3/files#diff-0838cb7940dc3fe4fed823789a42af475c1feebcabd7a9438d2eae3b7241ce93 I was able to build a Docker Image that involved compiling gflags with musl. That's not to say that compiling OpenJDK wouldn't be difficult, but this seems like a promising start...